go.mod

```diff
@@ -1,26 +1,18 @@
 module github.com/gruntwork-io/terragrunt
 
-go 1.12
+go 1.13
 
 require (
```

Comment:
why are we fully specifying the versions here?

Reply:
It's how go.mod works. Versions are bumped automatically with dependabot: https://dependabot.com/